config/schedule.rb

```diff
@@ -27,3 +27,8 @@
 every 1.day, at: '2:30 am', roles: [:prod] do
   rake " lib_jobs:alma_invoice_status_updates"
 end
+
+# Run on production at 2:30 am EST or 1:30 am EDT (after the records are published at 12 am)
+every 1.day, at: '6:30 am', roles: [:prod] do
```

Comment:
@sandbergja can you add one more comment that currenlty `dss-prod` is in UTC and 6:30am is UTC ?

Reply:
Yes, good idea!  Just added another comment.